pkgs/by-name/c-/c-intro-and-ref/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromSavannah,
+  texinfo,
+  texliveBasic,
+  ghostscript,
+  nix-update-script,
+}:
+let
+  rev = "62962013107481127176ef04d69826e41f51313c";
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "c-intro-and-ref";
+  version = "0.0-unstable-2024-08-31";
+
+  src = fetchFromSavannah {
+    repo = "c-intro-and-ref";
+    inherit rev;
+    hash = "sha256-Fmli3x8zvPntvCvV/wbEkxWzW9uDMZgCElPkKo9TS6Y=";
+  };
+
+  nativeBuildInputs = [
+    texinfo
+    ghostscript
+    texliveBasic
+  ];
+
+  buildFlags = [
+    "c.info"
+    "c.pdf"
+    # "c.doc" # FIXME: Not a HASH reference at (texinfo)/share/texinfo/Texinfo/Convert/DocBook.pm
+    "c.html"
+    "c.html.d"
+    "c.txt"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    dst_info=$out/share/info
+    dst_doc=$out/share/doc/c-intro-and-ref
+    mkdir -p $dst_info
+    mkdir -p $dst_doc
+
+    cp -prv -t $dst_info \
+        c.info c.info-*
+    cp -prv -t $dst_doc \
+        c.dvi c.pdf \
+        c.html \
+        c.html.d \
+        c.txt
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [ "--version=branch" ];
+  };
```

`nix-update-script` does not work on my machine.  However, this works:
```suggestion
  passthru.updateScript = unstableGitUpdater { url = "https://git.savannah.nongnu.org/git/c-intro-and-ref.git"; };
```

We can remove the `url = ...` once https://github.com/NixOS/nixpkgs/pull/352067 is merged.

---

done